src/io/aviso/taxi_toolkit/ui_map.clj

```diff
@@ -85,7 +85,8 @@
    (if (and (= 1 (count el-path))
             (string? (first el-path)))
      (find-with-f f (first el-path))
-     (f (xpath-or-css (resolve-element-path ui el-path))))))
+     (do
```

Comment:
This will be in a conflict with changes from PR-13. Fortunately, `(do ...)` is not necessary here.